muzei-api/build.gradle

```diff
@@ -36,7 +36,7 @@ allprojects { ext."signing.secretKeyRingFile" = props["signing.secretKeyRingFile
 
 dependencies {
     api "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
-    api "androidx.core:core:1.1.0"
+    api "androidx.core:core-ktx:1.3.2"
```

Comment:
This is intentionally `core:1.1.0` so that to avoid forcing unnecessary updates to Core in apps that depend on the `muzei-api` (Muzei API uses the minimal version of Core that it needs). Please revert this change.